packages/flutter/test/widgets/focus_traversal_test.dart

```diff
@@ -1128,14 +1322,10 @@ void main() {
       await tester.sendKeyEvent(LogicalKeyboardKey.arrowUp);
       expect(focusNodeUpperLeft.hasPrimaryFocus, isTrue);
     });
-    testWidgets('Focus traversal does not break when no focusable is available on a MaterialApp',   (WidgetTester tester) async {
+    testWidgets('Focus traversal does not break when no focusable is available on a MaterialApp', (WidgetTester tester) async {
```

Comment:
iOS doesn't have keyboard support yet, and the Cupertino widgets don't have any focus highlights, and most don't have focus nodes yet, so no, it's probably not worth it for `CupertinoApp`.

Since `MaterialApp` is built on `WidgetsApp`, it's also testing that, but I suppose I could add a separate test in case they ever get separated.